recipes/bacnet-stack/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "677f528aa46bebe5b7a129342b07fc491acd32b1":
```

```suggestion
  "20200218":
```
The policy of this repo is to use the date when the commit is made.
Actually, it is to use the latest commit on the specified date.